shimmer/learn/maintenance.md

```diff
@@ -0,0 +1,35 @@
+---
+id: maintenance
+description: What it actually takes to run the Shimmer network.
+image: /img/iota-wiki.png
+keywords:
+  - explanation
+  - shimmer
+  - proof of work
+  - node incentives
+  - storage deposit
+---
+
+# Maintaining Shimmer
+
+At its core, Shimmer needs little resources to run. To make a transaction, you need to cryptographically sign a short sequence of data and send it to a node to turn it into a block and progapate the block through the network. When another node receives that block, it runs it through a series of simple checks like verifying the signatures and checking what blocks it refers to, and add a new entry to its database of blocks. An [average personal computer](https://wiki.iota.org/hornet/getting_started#recommended-requirements) is enough to run a Shimmer node.
+
+## Incentives
+
+Fundamentally, Shimmer regulates only the communication between its nodes, not the nodes and the end users, and a random node has no incentives to take your transaction into work. There are public nodes that process strangers' transactions out of good spirit, but they can always go offline or put your transaction to the very end of the queue. If you need a reliable connection to the Tangle, the only way to get it is to run your own Shimmer node.
+
+If you are a casual user and make one or two transactions per day, then this would not be enough to convince you: if your transaction does not come through, you can wait for another few seconds or try a different node. But if you run a local business that accepts payments in $SMR tokens, or a cryptocurrency exchange, or a smart contract chain, then you need your Shimmer connection as reliable as possible, and you will run your own node.
```

Comment:
```suggestion
Fundamentally, Shimmer regulates only the communication between its nodes, not the nodes and the end users, and a random node has no incentives to take your transaction into work. Public nodes process strangers' transactions out of good spirit, but they can always go offline or put your transaction to the very end of the queue. If you need a reliable connection to the Tangle, the only way to get it is to run your own Shimmer node.

If you are a casual user and make one or two daily transactions, this would not be enough to convince you: if your transaction does not come through, you can wait for another few seconds or try a different node. But suppose you run a local business that accepts payments in $SMR tokens, a cryptocurrency exchange, or a smart contract chain. In that case, you need your Shimmer connection as reliable as possible, and you will run your own node.
```